google/resource_compute_instance_template.go

```diff
@@ -524,6 +530,7 @@ func resourceComputeInstanceTemplateCreate(d *schema.ResourceData, meta interfac
 	instanceProperties.CanIpForward = d.Get("can_ip_forward").(bool)
```

Comment:
totally up to you, but while you're here I wouldn't mind having these set inline in the struct:
```
instanceProperties := &computeBeta.InstanceProperties{
// stuff
}
```